src/test/contentMetaFilterSpec.js

```diff
@@ -0,0 +1,255 @@
+var configUtil = require('../libs/sb-config-util')
+var _ = require('lodash')
+
+var contentMetaConfig = require('./contentMetaConfig')
+
+var allowedChannels = contentMetaConfig.allowedChannels
```

Comment:
why do we need to create separate variables here can't we use directly the value from the constant

Reply:
i use allowedChannels in more than 10 places, so its better to assign it to a value